rules/integrations/aws/credential_access_aws_iam_assume_role_brute_force.toml

```diff
@@ -16,7 +16,62 @@ index = ["filebeat-*", "logs-aws*"]
 language = "kuery"
 license = "Elastic License v2"
 name = "AWS IAM Brute Force of Assume Role Policy"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating AWS IAM Brute Force of Assume Role Policy
+
+An IAM role is an IAM identity that you can create in your account that has specific permissions. An IAM role is similar
+to an IAM user, in that it is an AWS identity with permission policies that determine what the identity can and cannot
+do in AWS. However, instead of being uniquely associated with one person, a role is intended to be assumable by anyone
+who needs it. Also, a role does not have standard long-term credentials such as a password or access keys associated
+with it. Instead, when you assume a role, it provides you with temporary security credentials for your role session.
+
+Attackers may attempt to enumerate IAM roles in order to determine if a role exists before attempting to assume or
+hijack the discovered role.
+
+#### Possible investigation steps
+
+- Identify the user account that performed the action and whether it should perform this kind of action.
+- Verify if the `RoleName` parameter contains a unique value in all requests or if the activity is potentially a brute
+force attack.
+- Verify if the user account successfully updated a trust policy in the last 24 hours.
+- Examine whether this role existed in the environment by looking for past occurrences in your logs.
+- Investigate other alerts associated with the user account during the past 48 hours.
+- Contact the account and resource owners and confirm whether they are aware of this activity.
+- Consider the time of day. If the user is a human (not a program or script), did the activity take place during a normal
+time of day?
+- Examine the account's commands, API calls, and data management actions in the last 24 hours.
+- If you suspect the account has been compromised, scope potentially compromised assets by tracking servers, services,
+and data accessed by the account in the last 24 hours.
+
+### False positive analysis
+
+- Verify whether a role, if the unique observed on the failed attempts, existed in the environment. Automations can
```

Comment:
```suggestion
- Automations can Verify the roles targeted in the failed attempts, and whether the subject role previously existed in the environment. If only one role was targeted in the requests and that role previously existed, it may be a false positive, since automations can continue targeting roles that existed in the environment in the past and cause false positives (FPs).
```